src/core/lib/iomgr/udp_server.h

```diff
@@ -58,12 +58,11 @@ typedef void (*grpc_udp_server_orphan_cb)(grpc_exec_ctx *exec_ctx,
                                           grpc_fd *emfd);
 
 /* Create a server, initially not bound to any ports */
-grpc_udp_server *grpc_udp_server_create(void);
+grpc_udp_server *grpc_udp_server_create(struct grpc_server *server);
```

Comment:
Why does this need a server pointer?

This is mixing stack levels... definitely against this change.